cardano-api/gen/Gen/Cardano/Api/Typed.hs

```diff
@@ -170,8 +175,8 @@ genScriptData =
     genInteger :: Gen Integer
     genInteger = Gen.integral
                   (Range.linear
-                    (-fromIntegral (maxBound :: Word64) :: Integer)
-                    ( fromIntegral (maxBound :: Word64) :: Integer))
```

Comment:
What's the reason for this change?

Reply:
I undid this. There was a test failure that is no longer appearing.